dw-al2-concourse-ami/dw-concourse-ami-install.sh

```diff
@@ -2,8 +2,8 @@
 set -eEu
 
 # Set Proxy
-echo "HTTP_PROXY=$HTTP_PROXY"
-echo "HTTPS_PROXY=$HTTPS_PROXY"
+echo "HTTP_PROXY=$http_proxy"
+echo "HTTPS_PROXY=$https_proxy"
```

Comment:
```suggestion
echo "HTTP_PROXY=$HTTP_PROXY"
echo "HTTPS_PROXY=$HTTPS_PROXY"
```

These are debug lines, so should be stating what the correctly-cased variables are set to, otherwise confusion may  occur when HTTPS_PROXY is set to something different to https_proxy on its way into the script, but this debug statement would have you believe they're set to the same value.

Reply:
they are set to the case the env vars are set to.  I didn't fancy debugging why it's only set as lowercase.